definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -14670,5 +14670,33 @@
     "CaseFieldID": "ClarificationDigital",
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "bulkScanCaseReference",
+    "UserRole": "caseworker-divorce-systemupdate",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "bulkScanCaseReference",
+    "UserRole": "caseworker-divorce-bulkscan",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "bulkScanCaseReference",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "bulkScanCaseReference",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
```

Comment:
This is fine, but from the working perspective it shouldn't matter, because all updates happen through the API, which doesn't restrict access to fields.